encrypted-media-respec.html

```diff
@@ -3809,17 +3809,18 @@
         <section class="informative">
           <h4>Potential Attacks</h4>
           <p>Malicious pages could host legitimate applications in an iframe in an attempt hide an attack or deceive the user as to the source, such as making the use appear to be from a legitimate content provider.
-            This is especially relevent for user agents that support have <a href="#security-prompts">user alerts, prompts, and/or permissions</a>.
+            This is especially relevent for implementations that <a href="#security-consent">inform the user or require consent</a>, such as for security and/or <a href="#privacy-consent">privacy reasons</a>.
```

Comment:
Originally, we only had one "prompts" section, but now we have two. This text applies to both, so I don't want to only refer to the one in the security section. I'm not particularly happy with this solution, but it does the job. Suggestions welcome! This also applies to line 3819.